MapboxVisionAR/Render/ARScene/ConcreteNodes/ARLaneNode/ARLaneMesh/Error/ARMeshError.swift

```diff
@@ -0,0 +1,7 @@
+/// The error that occurred during creation AR mesh.
+enum ARMeshError: LocalizedError {
+    /// Mesh file isn't be found. Parametrized with name of resource file that contains mesh.
```

Comment:
Maybe it's better to make the enum conform to `Error` and state that info in localized description of `LocalizedError` protocol?

Reply:
Updated.